### DISCUSSION TITLE
Syntax highlighting for PEP 695: Type Parameter Syntax

### DISCUSSION THREAD

**Discussion by einarwar** [microsoft/pylance-release#5540] [Q&A] [answered]
I recently started using python 3.12, but to my surprise the syntax highlighting for the new type parameter added in PEP 695 was white! Generally white color is used when accessing attributes/methods on `Unknown` objects. See image below:

![image](https://github.com/microsoft/pylance-release/assets/21360860/f089811a-a323-497f-8695-af587922204f)

Is this color intentional or have i not set up pylance correctly?

**Comment by rchiodo** [comment]
I don't believe it's supported as a separate token yet:

![image](https://github.com/microsoft/pylance-release/assets/19672699/2c7bcfd4-3c9a-475c-bb03-c9e057f4c4bf)

**Comment by rchiodo** [comment, marked as answer]
Transfering this to an issue. We should add a new token for this I believe.

**Comment by rchiodo** [comment]
Created this issue:
https://github.com/microsoft/pylance-release/issues/5541
